aptos-move/aptos-vm/src/block_executor/mod.rs

```diff
@@ -272,7 +272,7 @@ impl BlockExecutorTransactionOutput for AptosTransactionOutput {
         aggregator_v1_writes: Vec<(StateKey, WriteOp)>,
         patched_resource_write_set: Vec<(StateKey, WriteOp)>,
         patched_events: Vec<ContractEvent>,
-    ) {
+    ) -> Result<(), PanicError> {
```

Comment:
This function `incorporate_materialized_txn_output` is declared/defined/called in a few other places. We have to update the function signature there as well right?